cfgov/unprocessed/css/forms.less

```diff
@@ -650,21 +441,315 @@
+              <span class="cf-input_text">Yes</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="yes">
+              <span class="cf-input_radio"></span>
+          </label>
+          <br>
+          <label class="cf-input_label" for="no">
+              <span class="cf-input_text">No</span>
+              <input class="cf-input cf-input_hidden" type="radio" name="cf-input-example_default" id="no">
+              <span class="cf-input_radio"></span>
+          </label>
+        codenotes:
+          - |
+            Structural cheat sheet:
+            -----------------------
+            label.cf-input_label
+              span.cf-input_text
+              input[type=radio].cf-input.cf-input_hidden
+              span.cf-input_radio
+        notes:
+          - "Use the markup structure above to add radio button .cf-input elements."
+          - "Radio buttons require extra markup to apply custom styles."
+    tags:
+      - cf-forms
+*/
+
+@input-active:                  @pacific;
+
+@input-bg-disabled:             @gray;
+@input-border-disabled:         @gray-50;
+
+// Insets
+@input-inset-bg:                @gray-20;
+@input-inset-selected:          @pacific-20;
+
+
+.cf-input {
+
+    @checkbox-width: unit( 18px / @base-font-size-px, em);
+    @radio-width: unit( 17px / @base-font-size-px, em);
+
+    &[type="checkbox"],
+    &[type="radio"]{
+        .u-visually-hidden();
+
+        .lt-ie9 & {
+            border: 0;
+            float: left;
+            margin: .4em .4em 0 0;
+            position: static;
+            width: auto;
+        }
+    }
+
+    &[type="checkbox"] + label,
+    &[type="radio"] + label {
+        cursor: pointer;
+        margin-bottom: 0.5em;
+    }
+
+    &[type="checkbox"] + label::before,
+    &[type="radio"] + label::before {
+        background: white;
+        content: '\a0';
+        display: inline-block;
+        margin-top: unit( 2px / @base-font-size-px, em);
+        margin-right: .6em;
+    }
+
+    &[type="radio"] + label::before {
+        height: @radio-width;
+        width: @radio-width;
+        border-radius: 100%;
+        box-shadow: 0 0 0 2px #fff, 0 0 0 3px @input-border;
+    }
+
+    &[type="radio"]:checked + label::before {
+        background-color: @input-active;
+        box-shadow: 0 0 0 2px @input-bg, 0 0 0 3px @input-active;
+    }
+
+    &[type="radio"]:hover + label::before {
+        box-shadow: 0 0 0 2px @input-bg, 0 0 0 3px @input-active;
+    }
+
+    &[type="radio"]:focus + label::before {
+        box-shadow: 0 0 0 1px @input-bg, 0 0 0 3px @input-active;
+    }
+
+    &[type="checkbox"] + label::before {
+        height: @checkbox-width;
+        width: @checkbox-width;
+        box-shadow: 0 0 0 1px @input-bg, 0 0 0 2px @input-border;
+    }
+
+    &[type="checkbox"]:checked + label::before {
+        content: @cf-icon-approved;
+        font-family: 'CFPB Minicons';
```

Comment:
I tried that first. It doesn't quite work since it adds some extra styles that break this implementation.